plugin/src/lib.rs

```diff
@@ -724,6 +724,23 @@ fn fold_first_block(block: P<Block>, m: &mut MutatorPlugin) -> P<Block> {
                         .unwrap(),
                 );
             }
+            let (ref key, ref values) = for (key, &value) in interchangeables.iter() {
+                let n = *current_count;
+                let ident = key.to_ident();
+                add_mutations(
+                    cx,
+                    mutations,
```

Comment:
@llogiq - Is what I'm doing here, correct? I'm not sure how to call `add_mutations` with single mutation.

Reply:
Almost – the message should read `&format!("exchange {} with {}", ident.as_str(), value_ident.as_str())`. For that you also need to get the ident of the value (same as with the key).